tools/klee/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+add_executable(klee
+	main.cpp
+)
+
+set(LLVM_COMPONENTS
+	bitreader
+	bitwriter
+	engine
+	ipo
+	irreader
+	jit
+	linker
+	support
+)
+
+if (${LLVM_PACKAGE_VERSION} VERSION_LESS "3.5")
+	llvm_map_components_to_libraries(LLVM_LIBS ${LLVM_COMPONENTS})
+else()
+	llvm_map_components_to_libnames(LLVM_LIBS ${LLVM_COMPONENTS})
+endif()
+
+target_link_libraries(klee kleeCore kleeBasic kleeModule kleaverSolver kleaverExpr kleeSupport ${LLVM_LIBS})
+
+target_link_libraries(klee stp)
```

Comment:
Linking against STP needs to be guarded by a variable indicating if STP is being used.